central/plugins/org.jboss.tools.central/src/org/jboss/tools/central/editors/SoftwarePage.java

```diff
@@ -108,7 +107,7 @@ public class SoftwarePage extends AbstractJBossCentralPage implements IRunnableC
 	
 	public SoftwarePage(FormEditor editor) {
 		super(editor, ID, "Software/Update");
-		monitor = new DelegatingProgressMonitor();
+		monitor = new org.jboss.tools.foundation.core.jobs.DelegatingProgressMonitor();

```

Comment:
Why is this form used ? Can't we use the short class name ?

Reply:
mylyn migration leftover